core/src/col_metadata.rs

```diff
@@ -104,7 +104,7 @@ pub struct VersionedJsonSchema {
 }
 
 impl SqlGetSchemaResponse {
-    /// Converts a sqlGetResultSchema command response into a list of column
```

Comment:
These comments were a result of regex replace, but I like them better this way slightly, hah!